src/components/opt-in-out/index.tsx

```diff
@@ -25,16 +26,16 @@ export const PLATFORM_OPTIONS: PlatformOptionRedirectData = {
   },
   'waypoint-io': {
     base_url: 'https://www.waypointproject.io/',
-    getRedirectPath: function TODO() {
-      return 'test'
+    getRedirectPath(path) {
+      return this.base_url + getIoRedirectPath(path)
     },
     cookieKey: 'waypoint-io-beta-opt-in',
     cookieAnalyticsKey: 'waypoint-io-beta-opt-in-tracked',
   },
   'vault-io': {
     base_url: 'https://www.vaultproject.io/',
-    getRedirectPath: function TODO() {
-      return 'test'
+    getRedirectPath(path) {
+      return this.base_url + getIoRedirectPath(path)
```

Comment:
We'll still need to [add the opt in component ](https://github.com/hashicorp/dev-portal/blob/main/src/views/product-tutorials-view/index.tsx#L76)to the docs layouts, not sure if you wanted to handle that on a different PR.

Reply:
Yep! Planning to add that in a follow-up 👍

Reply:
Oh just watched you vid, saw that you want to do that in a separate PR, please disregard 😄